Exception.php

```diff
@@ -55,10 +55,10 @@ class Exception extends Idle implements Event\Source
      * array that contains the list of formatted string for the message. If
      * chaining, we can add a previous exception.
      *
-     * @param   string          $message      Formatted message.
-     * @param   int             $code         Code (the ID).
-     * @param   array           $arguments    Arguments to format message.
-     * @param   \BaseException  $previous     Previous exception in chaining.
+     * @param   string      $message      Formatted message.
+     * @param   int         $code         Code (the ID).
+     * @param   array       $arguments    Arguments to format message.
+     * @param   \Exception  $previous     Previous exception in chaining.
```

Comment:
`BaseException` only exists on non-stable versions of php7 ([rfc](https://wiki.php.net/rfc/throwable-interface), [manual](http://php.net/manual/en/language.errors.php7.php))